spacer/tasks.py

```diff
@@ -100,14 +100,15 @@ def classify_image(msg: ClassifyImageMsg) -> ClassifyReturnMsg:
 
     # Classify
     clf = load_classifier(msg.classifier_loc)
-    scores = [(row, col, clf.predict_proba(features.get_array((row, col))).
-               tolist()) for row, col in msg.rowcols]
+    scores = [(row, col,
+               clf.predict_proba(features.get_array((row, col))).tolist()[0])
+              for row, col in msg.rowcols]
 
     # Return
     return ClassifyReturnMsg(
         runtime=time.time() - t0,
         scores=scores,
-        classes=list(clf.classes_),
+        classes=clf.classes_.tolist(),
```

Comment:
Cool! This should solve that JSON serialization issue for CoralNet's LocalQueue.

Reply:
Yups. That was all my bad.